model/channel_member.go

```diff
@@ -10,22 +10,25 @@ import (
 )
 
 const (
-	CHANNEL_ROLE_ADMIN     = "admin"
-	CHANNEL_NOTIFY_ALL     = "all"
-	CHANNEL_NOTIFY_MENTION = "mention"
-	CHANNEL_NOTIFY_NONE    = "none"
-	CHANNEL_NOTIFY_QUIET   = "quiet"
+	CHANNEL_ROLE_ADMIN          = "admin"
+	CHANNEL_NOTIFY_DEFAULT      = "default"
+	CHANNEL_NOTIFY_ALL          = "all"
+	CHANNEL_NOTIFY_MENTION      = "mention"
+	CHANNEL_NOTIFY_NONE         = "none"
+	CHANNEL_MARK_UNREAD_ALL     = "all"
+	CHANNEL_MARK_UNREAD_MENTION = "mention"
 )
 
 type ChannelMember struct {
-	ChannelId    string `json:"channel_id"`
-	UserId       string `json:"user_id"`
-	Roles        string `json:"roles"`
-	LastViewedAt int64  `json:"last_viewed_at"`
-	MsgCount     int64  `json:"msg_count"`
-	MentionCount int64  `json:"mention_count"`
-	NotifyLevel  string `json:"notify_level"`
-	LastUpdateAt int64  `json:"last_update_at"`
+	ChannelId       string `json:"channel_id"`
+	UserId          string `json:"user_id"`
+	Roles           string `json:"roles"`
+	LastViewedAt    int64  `json:"last_viewed_at"`
+	MsgCount        int64  `json:"msg_count"`
+	MentionCount    int64  `json:"mention_count"`
+	NotifyLevel     string `json:"notify_level"`
+	MarkUnreadLevel string `json:"mark_unread_level"`
```

Comment:
hmm...I need to think through this more.  Seems weird to have 2 cols doing almost the same thing.  I would think about making NotifyLevel a prop bag (I think it might have been impl like this once).